README.md

```diff
@@ -1 +1,9 @@
-# hyfer
\ No newline at end of file
+## Hyfer
+
+This is a small setup for the project. `react-router-dom` is added and configured so please take a look in `app.js` to how it all work.
```

Comment:
First of all, you are correct in that you only need `react-router-dom` and not `react-router` as of v4. Check out [this issue](https://github.com/ReactTraining/react-router/issues/4648) to clarify any confusion.

> You'll use the bindings for your environment. You will probably want to use react-router-dom for 99.9% of web dev.

Reply:
Ah nice that clears things a bit.